parts/k8s/addons/blobfuse-flexvolume.yaml

```diff
@@ -21,6 +21,15 @@ spec:
 {{- end}}
     spec:
       priorityClassName: system-cluster-critical
+      affinity:
```

Comment:
@invidian adding these declarations instead of not allowing the addon to be installed in a flatcar scenario, as we support heterogenous node pool configurations